### PR TITLE
Upgraded to full Symfony 3.3 installation.

### PR DESCRIPTION
**Pull request by GregHolmes**
Hi,

I hope you don't mind me doing this, but I found inconsistencies with your repository regarding Symfony 3.3. I forked your repo, installed a new set up and implemented the same changes you had. But this is running all Symfony 3.3 from a fresh install. Removing I believe legacy code and technical debt.

### CODE REVIEW

File: 00-Starter-Seed/app/config/routing.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ hwi_oauth_login:
     prefix:   /login
 
 auth0_login:
-    path: /auth0/callback
+    path:    /auth0/callback
```
Comment (lukeoliff) [inline 00-Starter-Seed/app/config/routing.yml]: You've been removed whitespace (which is nice) but you've added some here.

Comment (GregHolmes) [reply 00-Starter-Seed/app/config/routing.yml]: Thanks, removed it.

**Comment by GregHolmes** [comment]
@alexisluque  what do you think about this? :)

**Comment by GregHolmes** [comment]
Sorry @alexisluque which one?  You've said "Change the line 21 in Dockerfile for this one:" but nothing following.

File: 00-Starter-Seed/composer.json [hunk 2]
```diff
@@ -4,62 +4,69 @@
...
-            "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget"
+        "post-install-cmd": [
+            "@symfony-scripts"
         ],
-        "build-params": [
```
Comment (alexisluque) [inline 00-Starter-Seed/composer.json]: This script is also necessary, is used in docker execution.

**Comment by alexisluque** [comment]
Hi @GregHolmes, I edited the comment

**Comment by alexisluque** [comment]
@GregHolmes, I'm sorry, in `Auth0ResourceOwner.php` is line 42 instead.

**Comment by GregHolmes** [comment]
@alexisluque Very sorry for the delay, I've resolved this issues you've brought up. 
If it's ok with you, I'll work on getting this updated to Symfony 4 in the next few weeks. I currently have 4 articles in PR with the blog mostly using Symfony 4.

**Comment by aaguiarz** [comment]
@GregHolmes Thanks Greg! About Symfony 4, do you think it should be our primary platform for the Quickstart? Should we keep it in a separate branch?

@alexisluque let's do a final review, make sure the changes are reflected in the Quickstart doc if necessary

**Comment by GregHolmes** [comment]
@aaguiarz  You can choose. I'm happy to do :)

**Pull request merged**